### PR TITLE
Removed warnings on tests and start. Added package-lock for npm5 support

### PR DESCRIPTION
**Pull request by juampick**
## Background
_Executing the project have some warnings on eslint, and tests._
## Changes done
- Updated components to remove warnings.
- Added `package-lock` for npm5 support.

## Pending to be done
Hopefully today I'll be doing a PR with libraries updates.
## Notes
N/A

### CODE REVIEW

File: app/routes.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {
 } from "react-router";
 import App from "./components/App";
 import HomePage from "./components/home/HomePage";
-import UsersPage from "./components/users/UsersPage";
+import UsersPageComponent from "./components/users/UsersPage";
```
Comment (matiasbeckerle) [inline app/routes.js]: Not sure why this change but I liked the previous one 😄.

File: app/components/common/ConfirmModal.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ ConfirmModal.propTypes = {
   footer: PropTypes.string,
   confirm: PropTypes.func.isRequired,
   close: PropTypes.func,
-  modal: PropTypes.object.isRequired
+  modal: PropTypes.object
```
Comment (AlphaGit) [inline app/components/common/ConfirmModal.js]: Just double checking: is this property not required anymore? It seems like the only purpose of this component is to display this modal.

File: app/components/common/Modal.js [hunk 3]
```diff
@@ -66,7 +66,7 @@ Modal.propTypes = {
   body: PropTypes.string,
   footer: PropTypes.string,
   close: PropTypes.func,
-  modal: PropTypes.object.isRequired
+  modal: PropTypes.object
```
Comment (AlphaGit) [inline app/components/common/Modal.js]: Just double checking: is this property not required anymore? It seems like the only purpose of this component is to display this modal.

File: app/routes.js (hunk 1 above)
Comment (AlphaGit) [inline app/routes.js]: I agree with this change, but should we change all referenced components so that we don't introduce an inconsistency?

File: app/services/apiEndpoints.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-export const BASE_URL = API_BASE_URL;
+export const BASE_URL = API_BASE_URL; // eslint-disable-line no-undef
```
Comment (AlphaGit) [inline app/services/apiEndpoints.js]: Is there any chance we can import the variable instead of referencing it globally?

File: app/routes.js (hunk 1 above)
Comment (juampick) [reply app/routes.js]: This is not only a naming thing. This was throwing a warning. Why? because UsersPage is a decorated component using redux. As the comment on the code test: 
`Using the undecorated component (In order to be able to test the component itself without having to deal with the decorator)`. 
So that the component have 2 exports, the default decorated and the undecorated without redux, only react. To be able to import correctly without warnings, we need to change the name of the imported component when using it (ie: routes). This is the same that we applied on VAS. Thanks

File: app/components/common/ConfirmModal.js (hunk 2 above)
Comment (juampick) [reply app/components/common/ConfirmModal.js]: This was throwing a warning after some updates for the team. Not sure. But you can review it later. Thanks

File: app/components/common/Modal.js (hunk 3 above)
Comment (juampick) [reply app/components/common/Modal.js]: Same as above.

File: app/routes.js (hunk 1 above)
Comment (juampick) [reply app/routes.js]: Nope.. please check my answer to Mati's question.

File: app/services/apiEndpoints.js (hunk 4 above)
Comment (juampick) [reply app/services/apiEndpoints.js]: Not sure, I don't want to change logic in the PR. Only upgrade of react, removing warnings, and that kind of tasks. Maybe can be reviewed later all the others improvements. Thanks

File: app/routes.js (hunk 1 above)
Comment (AlphaGit) [reply app/routes.js]: @juampick I didn't know that! Thanks for sharing.

BTW, I redacted part of your comment. Remember this is a public repository and we cannot share the specifics of what we do for each of our clients.

Comment (juampick) [reply app/routes.js]: You are right. Thanks JD.
